changelogs/fragments/4049-profile-for-scaleway-inventory.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway plugin inventory - Add profile parameter, ``scw_profile`` (https://github.com/ansible-collections/community.general/pull/4049).
```

Comment:
```suggestion
  - scaleway inventory plugin - add profile parameter ``scw_profile`` (https://github.com/ansible-collections/community.general/pull/4049).
```

Reply:
I asked myself the same question and didn't find better solution between the both.
So I'm inspired from the aws inventory plugin and use the same syntax